Laravel/base/composer.json

```diff
@@ -2,54 +2,38 @@
     "name": "laravel/laravel",
     "type": "project",
     "description": "The Laravel Framework.",
-    "keywords": [
-        "framework",
-        "laravel"
-    ],
+    "keywords": ["framework", "laravel"],
     "license": "MIT",
     "require": {
-        "php": "^7.2.5",
-        "bref/bref": "^0.5.25",
-        "bref/laravel-bridge": "^0.2.0",
-        "fideloper/proxy": "^4.2",
-        "fruitcake/laravel-cors": "^1.0",
-        "guzzlehttp/guzzle": "^6.3",
-        "laravel/framework": "^7.0",
-        "laravel/tinker": "^2.0"
+        "php": "^7.3|^8.0",
+        "bref/bref": "^1.2",
```

Comment:
With that upgrade the `runtime` option in serverless.yml needs to be updated too: https://bref.sh/docs/news/01-bref-1.0.html#amazon-linux-2